contracts/examples/bonding-curve/src/bc_function.rs

```diff
@@ -0,0 +1,50 @@
+elrond_wasm::imports!();
+elrond_wasm::derive_imports!();
+
+#[derive(TopEncode, TopDecode, NestedEncode, NestedDecode, TypeAbi, PartialEq, Clone)]
+pub enum SupplyType {
+	Limited,
+	Limitless,
```

Comment:
Suggest renaming Limitless to Unlimited.

Limitless denotes something like potential, while unlimited denotes more of a numerical concept.